lib/components/tabs.js

```diff
@@ -26,6 +26,7 @@ export default class Tabs extends Component {
             <div className={css('title')}>{tabs[0].title}</div> :
             [
               <ul
+                key="tabs-container"
```

Comment:
`tabs-list` also goes along with the `list` class name that is being added to it :smile: